### ISSUE TITLE
Icon for Microsoft Edge (Chromium) not shown in Open Shell on Windows 11

### ISSUE THREAD

**Issue by anonimuos** [Open-Shell/Open-Shell-Menu#793]
App icons for all other apps are shown in Open Shell on Windows 11 but the icon for Edge Chromium is not shown. It shows a generic icon instead.

**Comment by bonzibudd** [comment]
The "Apps" folder in Open-Shell only shows UWP/MS Store apps, which doesn't include Microsoft Edge. Therefore, Edge shows up in the main programs list instead. What you're probably seeing is the remnant shortcut for the old Edge app, and that issue is being tracked by https://github.com/Open-Shell/Open-Shell-Menu/issues/294

**Comment by ale5000-git** [comment]
I have the icon in the "Apps" folder and it open the Edge (Chrome).
I have also deleted the normal shortcut in the main programs list and the one in the "Apps" folder still works.

**Comment by ale5000-git** [comment]
If I add it to OpenShell via right click I get this shortcut (that also works):
![shortchut](https://user-images.githubusercontent.com/15793015/127742258-9326dd43-154b-4fb3-89e9-41fb16dc0e04.png)

**Comment by anonimuos** [comment]
> The "Apps" folder in Open-Shell only shows UWP/MS Store apps, which doesn't include Microsoft Edge. Therefore, Edge shows up in the main programs list instead. What you're probably seeing is the remnant shortcut for the old Edge app, and that issue is being tracked by #294

No. In Windows 11, it shows even Notepad, Edge Chromium, Paint, Snipping Tool and Skype in the "Apps" folder since they are updated via the Store now.

**Comment by CRGinWindsor** [comment]
Does anybody have a solution to this???

**Comment by Neustradamus** [comment]
Linked to:
- https://github.com/coddec/Classic-Shell/issues/45
- https://github.com/Open-Shell/Open-Shell-Menu/issues/294
- https://github.com/Open-Shell/Open-Shell-Menu/issues/479
- https://github.com/Open-Shell/Open-Shell-Menu/issues/737
- https://github.com/Open-Shell/Open-Shell-Menu/issues/788
- https://github.com/Open-Shell/Open-Shell-Menu/issues/793
